commands/plugin/install.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
```

Comment:
Let's name it `plugin-install.js`  (we consider renaming all sub level commands, so they're joined with `-` and not whitespace)